articles/libraries/auth0js/index.md

```diff
@@ -7,22 +7,53 @@ url: /libraries/auth0js
 
-Auth0.js is a client-side library for [Auth0](http://auth0.com). It allows you to trigger the authentication process and parse the [JWT](http://openid.net/specs/draft-jones-json-web-token-07.html) (JSON web token) with just the Auth0 `clientID`. Once you have the JWT you can use it to authenticate requests to your http API and validate the JWT in your server-side logic with the `clientSecret`.
+Auth0.js is a client-side library for [Auth0](http://auth0.com). It allows you to trigger the authentication process and parse the [JSON Web Token](http://openid.net/specs/draft-jones-json-web-token-07.html) (JWT) with just the Auth0 `clientID`. Once you have the JWT, you can use it to authenticate requests to your HTTP API and validate the JWT in your server-side logic with the `clientSecret`.
 
-## Example
+## Ready-to-Go Example
 
-The example directory has a ready-to-go app. In order to run it you need [node](http://nodejs.org/) installed, download dependencies with `npm install`, then execute `npm run example` from the root of this project.
+The [example directory](https://github.com/auth0/auth0.js/tree/master/example) of the auth0.js library is a ready-to-go app that can help you to quickly and easily try out auth0.js. In order to run it, follow these quick steps:
+1. If you don't have [node](http://nodejs.org/) installed, do that now
+1. Download dependencies by running `npm install` from the root of this project
+1. Finally, execute `npm run example` from the root of this project, and then browse to your app running on the node server, presumably at `http://localhost:3000`.
+
+<img width="600" src="/media/articles/libraries/auth0js/auth0js-example.png" />
+
+It's that easy!
 
 ## Usage
 
-Take `auth0.js` or `auth0.min.js` from the `/build` directory and import it to your page.
+Now, let's get started integrating auth0.js into your project. We'll cover [methods of installation](#installation-options), [how to initialize auth0.js](#initialize), [signup](#signup), [login](#login), [Passwordless](#passwordless-authentication), [accessing user profiles](#user-profile), and more! 
 
-If you are using [browserify](http://browserify.org/) install with `npm i auth0-js --production --save`.
+### Installation Options
 
-> Note: The following examples use jQuery, but auth0.js is not tied to jQuery and any library can be used with it.
+You have a few options for using auth0.js in your project. Pick one of the below depending on your needs:
+
+Install via [npm](https://npmjs.org):
+
+```sh
+npm install auth0-js
+```
+
+Install via [bower](http://bower.io):
+
+```sh
+bower install auth0.js
+```
+
+Include via our CDN:
+
+```html
+<script src="${auth0js_url}"></script>
+```
+
+Or, just [download the project](https://github.com/auth0/auth0.js) and take `auth0.js` or `auth0.min.js` from the `/build` directory and import it to your page.
```

Comment:
Build is only present on github tags (for bower) so I'd remove this mention and let the customers use the other options.